lib/view_component/slotable_v2.rb

```diff
@@ -17,9 +17,19 @@ module SlotableV2
       # Hash of registered Slots
       class_attribute :registered_slots
       self.registered_slots = {}
+
+      class_attribute :_raise_on_deprecated_slot_setter
+      self._raise_on_deprecated_slot_setter = false
```

Comment:
```suggestion
      class_attribute :_raise_on_deprecated_slot_setter, default: false
```

Can we rely on the `default` parameter `class_attribute` accepts?

Reply:
I don't believe so, at least with the Ruby/Rails combinations we support.